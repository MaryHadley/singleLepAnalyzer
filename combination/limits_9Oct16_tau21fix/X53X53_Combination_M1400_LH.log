000 (1295367.000.000) 10/09 22:41:04 Job submitted from host: <192.168.63.249:41533>
...
001 (1295367.000.000) 10/09 22:41:26 Job executing on host: <192.168.63.228:44891>
...
006 (1295367.000.000) 10/09 22:41:34 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2272  -  ResidentSetSize of job (KB)
...
006 (1295367.000.000) 10/09 22:46:34 Image size of job updated: 283368
	143  -  MemoryUsage of job (MB)
	146416  -  ResidentSetSize of job (KB)
...
006 (1295367.000.000) 10/09 23:36:40 Image size of job updated: 283368
	143  -  MemoryUsage of job (MB)
	146428  -  ResidentSetSize of job (KB)
...
006 (1295367.000.000) 10/10 14:48:15 Image size of job updated: 297712
	156  -  MemoryUsage of job (MB)
	159352  -  ResidentSetSize of job (KB)
...
005 (1295367.000.000) 10/10 14:55:32 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 16:12:15, Sys 0 00:00:21  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 16:12:15, Sys 0 00:00:21  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1596  -  Run Bytes Sent By Job
	307  -  Run Bytes Received By Job
	1596  -  Total Bytes Sent By Job
	307  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :    31803        1   3824810
	   Memory (MB)          :      156     3072      3072
...
