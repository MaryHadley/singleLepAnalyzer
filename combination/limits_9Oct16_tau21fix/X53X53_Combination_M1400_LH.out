/var/lib/condor/execute/dir_24358
Total number of likelihood evaluations: 1100000101
Total number of likelihood evaluations: 11000101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1400_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1400_LH/quant-toys:0-sig-a66aff7c66.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1400_LH/quant-data-sig-e1313c17a3.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix/X53X53_Combination_M1400_LH
