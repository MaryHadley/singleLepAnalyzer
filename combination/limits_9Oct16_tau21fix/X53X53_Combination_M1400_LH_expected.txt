# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.078437    0.03981    0.16268  0.054552    0.11526
